crates/sui-bridge-cli/src/lib.rs

```diff
@@ -42,13 +42,20 @@ use sui_types::transaction::{ObjectArg, Transaction, TransactionData};
 use sui_types::{TypeTag, BRIDGE_PACKAGE_ID};
 use tracing::info;
 
+pub const SEPOLIA_BRIDGE_PROXY_ADDR: &str = "0xAE68F87938439afEEDd6552B0E83D2CbC2473623";
```

Comment:
is this really a constant? 
is it something we need to change for mainnet?
not super clear what this constant is doing here

Reply:
it's constant for each network so for mainnet there will be a new one.